### ISSUE TITLE
🚀 Feature: Helm chart to deploy on Kubernetes

### ISSUE THREAD

**Issue by ypicard** [novuhq/novu#2530]
### 🔖 Feature description

Documentation on how to deploy with Docker is available for Nova, but I think it would be relevant to also add a [Helm](https://helm.sh) chart to facilitate deployment on Kubernetes.

### 🎤 Why is this feature needed ?

There is no current streamlined way to deploy Novu on a Kubernetes cluster.

### ✌️ How do you aim to achieve this?

I would like Helm charts to be released along Nov releases for easier deployment on Kubernetes.

### 👀 Have you spent some time to check if this feature request has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Contributing Guidelines](https://github.com/novuhq/novu/blob/main/CONTRIBUTING.md)

### Are you willing to submit PR?

None

**Comment by titenkov** [comment]
I've created the Helm chart for Novu in the organization I'm working for right now 🙂 I'm happy to contribute and set it up if you will create a repository for helm charts in the organization

**Comment by scopsy** [comment]
@titenkov is it possible to add it as part of the monorepo we have to day in the docker folder?

**Comment by titenkov** [comment]
Yes, it's possible to keep it in the monorepo. It will require some non-code changes to the repo to make it possible to distribute (enabling GitHub pages at least).

You can check https://github.com/notifir/charts as an example (my background and journey towards start using Novu) 😅

**Comment by ypicard** [comment]
This would be great!

**Comment by ryshoooo** [comment]
@titenkov I'm about to create the helm chart in my organization as well. Let me know if you need help, I'd rather work on this open-sourcely rather than privately :)

**Comment by zek** [comment]
Any progress? I also started creating a helm chart.

**Comment by zek** [comment]
@scopsy I have prepared a chart. TBH It's my first helm chart. So it is not perfect :)
 It doesn't cover everything but I can create a PR and we can add missing parts with the help of community :)

I believe we should create a new repository for helm charts. What do you think?

**Comment by scopsy** [comment]
@zek we can create a new `helm` folder in the docker folder in the monorepo: https://github.com/novuhq/novu/tree/next/docker

What do you think?

**Comment by zek** [comment]
> https://github.com/novuhq/novu/tree/next/docker

It's okay for me. However we need to compile charts also.

**Comment by prajjwaldimri** [comment]
@zek Any update on the chart? I was thinking of creating a chart too but stumbled upon this issue.
